### DISCUSSION TITLE
multiple reader, multiple ripper

### DISCUSSION THREAD

**Discussion by Jovuf** [enzo1982/freac#334] [Help & Support]
hello, I definitely can't copy several cds at the same time, I have 5 cd players and when I insert my cds they all end up in the same list.
What interests me is to copy them all at the same time and in parallel.
I received a message concerning this idea two weeks ago but I did not know how to find the way to arrive there.
It would have been a matter of finding an option to uncheck which would be called "create a single playlist and/or cue sheet file per conversion" but I don't know where it is.

Also I found this option but it doesn't seem to be assigned anything:

configure general settings > Resource > parallel processing > enable parallel processing or enable super fast processing

But I don't get parallel processing :/

**Comment by Lee-Carre** [comment]
I think this is possible with newer versions (than of 2022-01-22).

Yes; answered in (your own duplicate) #321.
